loader/src/main/java/net/neoforged/fml/config/ConfigTracker.java

```diff
@@ -68,6 +98,33 @@ public void unloadConfigs(ModConfig.Type type) {
         this.configSets.get(type).forEach(this::closeConfig);
     }
 
+    public void openConfig(final ModConfig config, final Path configBasePath, @Nullable Path configOverrideBasePath) {
```

Comment:
I never liked this name. I would see open config as like a fileStream where I read a file contents. Maybe loadConfig would be a better name here? But you got loadConfig already below. hmmm

Reply:
The idea is that you might load the config multiple times, I suppose. We also have `closeConfig`. I suppose that `open`/`close` is more about the action of opening/closing the `FileConfig`, which might be `loaded` from multiple times over its lifetime?

Reply:
So it's more about when the file is being watched or not? `trackConfig` `untrackConfig`

Reply:
Kind of. It also controls when you can query config values.

Reply:
Idk of anything better than `trackConfig` `untrackConfig` but if you want to keep current name, you can resolve this convo